src/test/cpp/component.idl

```diff
@@ -192,4 +192,36 @@ namespace Component
             static event Handler Static;
         }
     }
+
+    namespace Params
+    {
+        [default_interface]
+        runtimeclass Class
+        {
+            Class();
+
+            void SyncCall(
+                String a,
+                Windows.Foundation.IReference<int> b,
+                Windows.Foundation.Collections.IIterable<int> c,
+                Windows.Foundation.Collections.IVectorView<int> d,
+                Windows.Foundation.Collections.IMapView<int, int> e,
+                Windows.Foundation.Collections.IVector<int> f,
+                Windows.Foundation.Collections.IMap<int, int> g);
+
+            Windows.Foundation.IAsyncAction NotSyncCall(
```

Comment:
curious name, why not AsyncCall?

Reply:
Because the old compiler looked for the substr "Async" and I wanted to make sure that would fail the new test as we now explicitly look for one of the the IAsyncXxx return type. I can probably clean up the naming now that this old code is gone. 👍